### PR TITLE
Update the Relations article

### PR DESCRIPTION
**Pull request by sofietoft**
## Description

Remove the section about how to create a Relation type in the backoffice, as this is no longer possible in Umbraco 14.

I also cleaned up the article a bit.

## Type of suggestion

* [ ] Typo/grammar fix
* [x] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

14+

### CODE REVIEW

File: 14/umbraco-cms/fundamentals/data/relations.md [hunk 1]
```diff
@@ -8,69 +8,42 @@ Umbraco sections are built around the concept of 'trees' and there is an implici
...
 
-We refer to these relationships in the manner of a 'Family Tree' - eg, One content item might be the 'Parent' of some content items, and those content items would be referred to as the 'Children' of that parent. Items within the same branch of the tree can also be described as 'Ancestors' or 'Descendants' of an item, depending if they are above or below them in the depth of the tree.
+We refer to these relationships in the manner of a 'Family Tree'. One content item might be the 'Parent' of some content items, and those items would be referred to as the 'Children' of that parent. Items within the same branch of the tree can also be described as 'Ancestors' or 'Descendants' of an item.
 
-There are methods in Umbraco to support querying content items by their relative position in the tree to the current page using these concepts, eg `Model.Ancestors()` or `Model.Children()` or `Model.Descendants()` - but what about 'Cousins'?
+There are methods available to support querying content items by their relative position to the current page. This is possible using the following concepts: `Model.Ancestors()`, `Model.Children()` or `Model.Descendants()`.
```
Comment (eshanrnh) [inline 14/umbraco-cms/fundamentals/data/relations.md:13]:
```suggestion
There are methods available to support querying content items by their relative position to the current page. This is possible using the following concepts: `Model.Ancestors()`, `Model.Children()`, or `Model.Descendants()`.
```

File: 14/umbraco-cms/fundamentals/data/relations.md [hunk 2]
```diff
@@ -8,69 +8,42 @@ Umbraco sections are built around the concept of 'trees' and there is an implici
...
 
 ## The Concept of Umbraco Relations
 
-Umbraco Relations allow you to relate almost any object in Umbraco to almost any other Umbraco object - under a defined _Relation Type_.
+Umbraco Relations allow you to relate almost any object in Umbraco to almost any other Umbraco object. This is done by defining a new _Relation Type_.
 
 ### How is this different to pickers?
 
-With a Content/Member/Media picker that you might add to a Content Type - the relationship is ONLY 1-way. Your content item knows it has 'picked' another content item, perhaps to display a shared banner across different pages, but that 'banner' content item doesn't know where it has been picked!
+With a Content, Member or Media picker the relationship only works as a 1-way street. The content item knows it has 'picked' another content item but that other content item does not know where it has been picked.
```
Comment (eshanrnh) [inline 14/umbraco-cms/fundamentals/data/relations.md:27]:
```suggestion
With a Content, Member, or Media picker the relationship only works as a 1-way street. The content item knows it has 'picked' another content item but that other content item does not know where it has been picked.
```

File: 14/umbraco-cms/fundamentals/data/relations.md [hunk 3]
```diff
@@ -8,69 +8,42 @@ Umbraco sections are built around the concept of 'trees' and there is an implici
...
-
-    Example: For relating Members to their uploaded Images, we might create a 'Member Images' relation
-
-    ![Member Images](../../../../10/umbraco-cms/fundamentals/data/images/member-images.png)
-6. Select **Is Dependency** as **Yes** if you wish this Relation Type to be used in Umbraco's 'tracking' functionality; that warns editors when trying to delete entities if they are 'used' in a Relation Type marked as 'Is Dependency'
-7. Click **Create** and you'll see your new Relation Type created in the Relation Types folder. You can see the 'Alias' that you'll need to make note of when working with Relations.
+## Viewing Relations
 
-    ![Member Images](../../../../10/umbraco-cms/fundamentals/data/images/relation-alias.png)
+It is possible to view the existing Relation Types from the Umbraco backoffice.
```
Comment (eshanrnh) [inline 14/umbraco-cms/fundamentals/data/relations.md:37]:
```suggestion
It is possible to view the existing Relation Types from the Umbraco backoffice:
```

File: 14/umbraco-cms/fundamentals/data/relations.md [hunk 4]
```diff
@@ -8,69 +8,42 @@ Umbraco sections are built around the concept of 'trees' and there is an implici
...
+Umbraco Relations works as a 2-way street. When creating a relation between two different types of entities, it will be possible to find one entity from the other and vice versa. As an example this provides the option to list out all the pages that a content banner had been picked on.
 
 ## Relation Types
 
-To create and use Relations in your Umbraco Website, you need to define a 'Relation Type' to specify what the two types of entities will be relatable and to give the Relation Type an alias. So, you can test if two items are related for a particular 'relation type'. Two items might be related under multiple different Relation Types, and you might be only interested in your 'Related Language Page' Relation Type.
+To create and use Relations in your Umbraco Website, you need to define a 'Relation Type'. The Relation Type will specify what two types of entities will be relatable. Two items might be related under multiple different Relation Types, and you might be only interested in your 'Related Language Page' Relation Type.
```
Comment (eshanrnh) [inline 14/umbraco-cms/fundamentals/data/relations.md:33]: Umm.. I don't think we can define a Relation Type anymore from the Backoffice?

**Review by eshanrnh — COMMENTED** [review]
Looks good to me, @sofietoft 🙌 Just have few minor comments.

File: 14/umbraco-cms/fundamentals/data/relations.md (hunk 4 above)
Comment (sofietoft) [reply 14/umbraco-cms/fundamentals/data/relations.md:33]: True 🤔 
I think I'll try to rephrase the entire section.

File: 14/umbraco-cms/fundamentals/data/relations.md [hunk 5]
```diff
@@ -8,84 +8,57 @@ Umbraco sections are built around the concept of 'trees' and there is an implici
...
 
-Umbraco Relations are 2-way, when you create a relation between two different types of entities, you can alway find one entity from the other and vice versa. eg you could list out in the backoffice all the pages that a content banner had been picked on.
+Umbraco Relations works as a 2-way street. When creating a relation between two different types of entities, it will be possible to find one entity from the other and vice versa. As an example this provides the option to list out all the pages that a content banner had been picked on.
 
 ## Relation Types
 
-To create and use Relations in your Umbraco Website, you need to define a 'Relation Type' to specify what the two types of entities will be relatable and to give the Relation Type an alias. So, you can test if two items are related for a particular 'relation type'. Two items might be related under multiple different Relation Types, and you might be only interested in your 'Related Language Page' Relation Type.
+A Relation Type specifies how two types of entities are be related. Two items might be related under multiple different Relation Types, and you might be only interested in your 'Related Language Page' Relation Type.
```
Comment (eshanrnh) [inline 14/umbraco-cms/fundamentals/data/relations.md:33]:
```suggestion
A Relation Type specifies how two types of entities are related. Two items might be related under multiple Relation Types, and you might only be interested in your 'Related Language Page' Relation Type.
```

File: 14/umbraco-cms/fundamentals/data/relations.md (hunk 4 above)
Comment (eshanrnh) [reply 14/umbraco-cms/fundamentals/data/relations.md]: Looks good to me.

**Pull request merged**